README.md

```diff
@@ -1152,8 +1159,6 @@ $ make clean
 $ DEBUG=true operator-sdk run --local --namespace=<namespace>
 ```
 
-Before submitting a [pull request](https://help.github.com/en/articles/about-pull-requests) to the Kogito Operator repository, review the instructions for [Contributing to the Kogito Operator](docs/CONTRIBUTING.MD).
```

Comment:
Why is this removed?

Reply:
This section is at the bottom of the article (just a couple of lines later). So, this line didn't match with the topic of the current section.

Reply:
where exactly?

Reply:
This paragraph should be in the "Development" session. Please move it there.

Reply:
What I was trying to say is that this topic is covered in the last section: https://github.com/kiegroup/kogito-cloud-operator#contributing-to-the-kogito-operator
which is saying the same but using other words.
What do you think?

Reply:
You're right! Thanks for the input.